test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
- [ ] You were requested not to use file generation command like scaffold command, but you used it anyways. Kindly remove this scaffold file.